src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,111 @@
+package duke;
+
+import duke.task.Task;
+import java.util.List;
+
+/**
+ * Interacts with the user by outputting certain responses
+ * based on user input
+ */
+public class Ui {
+    private final static String line = "______________________________________________________";
+    private final static String lineIndent = "    ";
+    private final static String listIndent = "       ";
+    private final static String textIndent = "     ";
```

Comment:
Since these are constants, it's recommended to have them all be uppercase (LINE, LINE_INDENT, ...). And perhaps that since they're associated with one another, they could also be prefixed with a common name (MSG_LINE, MSG_LINE_INDENT, ...)

Reply:
Yup I think both of you are right! Thanks for the suggestions!